### PR TITLE
chore: Spike/public path

### PR DESCRIPTION
**Pull request by TomWoodward**

### CODE REVIEW

File: jenkins/release.jenkins [hunk 1]
```diff
@@ -39,7 +42,8 @@ pipeline {
       }
       steps {
         sh "docker cp $BUILD_TAG:/code/build ."
-        sh "./bin/aws s3 sync --content-type 'text/html' build/ s3://${params.BUCKET}/${params.RELEASE_ID}/rex/releases"
+        sh "./bin/aws s3 sync --exclude 'books/*' build/ s3://${params.BUCKET}/rex/releases/${params.RELEASE_ID}"
+        sh "./bin/aws s3 sync --content-type 'text/html' build/books/ s3://${params.BUCKET}/rex/releases/${params.RELEASE_ID}/books"
```
Comment (TomWoodward) [inline jenkins/release.jenkins]: this is an unrelated change to stop setting the mime type to html for all files, the sync command will infer the mime type from the extension for other files, but the book pages don't have extensions

File: jenkins/release.jenkins [hunk 2]
```diff
@@ -15,8 +15,11 @@ pipeline {
       }
     }
     stage('build') {
+      environment {
+        PUBLIC_URL = "/rex/releases/${params.RELEASE_ID}"
+      }
       steps {
-        sh "docker build -t openstax/rex-web ."
+        sh 'docker build -t openstax/rex-web --build-arg PUBLIC_URL="$PUBLIC_URL" .'
```
Comment (TomWoodward) [inline jenkins/release.jenkins]: this is the actual change for the spike

File: script/prerender.tsx [hunk 3]
```diff
@@ -44,7 +44,7 @@ async function render() {
   const archiveLoader = createArchiveLoader(`http://localhost:${port}/contents/`);
 
   async function renderManifest() {
-    writeFile(path.join(ASSET_DIR, '/books/release.json'), JSON.stringify({
+    writeFile(path.join(ASSET_DIR, '/release.json'), JSON.stringify({
```
Comment (TomWoodward) [inline script/prerender.tsx]: moved this back to the root of the build folder because its easier to manage the files when everything in books is the content pages

File: jenkins/release.jenkins [hunk 4]
```diff
@@ -15,8 +15,11 @@ pipeline {
       }
     }
     stage('build') {
+      environment {
+        PUBLIC_URL = "/rex/releases/${params.RELEASE_ID}"
```
Comment (jpslav) [inline jenkins/release.jenkins]: why is it called `PUBLIC_URL`?

Comment (jpslav) [reply jenkins/release.jenkins]: oh ok - `PUBLIC_URL` just happens to be what react-scripts is expecting.

**Review by jpslav — DISMISSED** [review]

**Review by jpslav — APPROVED** [review]
